src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java

```diff
@@ -217,6 +229,23 @@ else if (targetY <= levelDepth && workerY > levelDepth)
 
                 //Then add the ladder position as the latest node.
                 proxyList.add(new BlockPos(ladderPos.getX(), level.getDepth(), ladderPos.getZ()));
+
+                if(level.getRandomNode().getParent() != null)
+                {
+                    List<BlockPos> nodesToTarget = new ArrayList<>();
+                    com.minecolonies.coremod.entity.ai.citizen.miner.Node currentNode = level.getNode(level.getRandomNode().getParent());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'com.minecolonies.coremod.entity.ai.citizen.miner.Node' due to existing import 'com.minecolonies.coremod.entity.ai.citizen.miner.*' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)